api/scripts/populateLocalDynamo.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-restricted-imports */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// Run yarn ts-node -r tsconfig-paths/register scripts/populateLocalDynamo.ts
+import { BatchWriteItemCommand } from "@aws-sdk/client-dynamodb";
+import { marshall } from "@aws-sdk/util-dynamodb";
+import { generateUserData } from "../../shared/src/test";
+import { CURRENT_VERSION } from "../../shared/src/userData";
+import { createDynamoDbClient } from "../src/db/config/dynamoDbConfig";
+import { DynamoBusinessDataClient } from "../src/db/DynamoBusinessDataClient";
+import { DynamoDataClient } from "../src/db/DynamoDataClient";
+import { DynamoUserDataClient } from "../src/db/DynamoUserDataClient";
+import { BUSINESSES_TABLE, USERS_TABLE } from "../src/functions/config";
```

Comment:
[dust] Can we make these absolute imports?

for example
```typescript
import { generateUserData } from "@shared/test";
```

Reply:
Just noting as a follow-up from our sync...

I was able to replace all of the relative imports with absolute imports and the code worked as expected.

Not a massive deal, but if we are intending to keep this around long-term it might be nice to include.
```typescript
import { createDynamoDbClient } from "@db/config/dynamoDbConfig";
import { DynamoBusinessDataClient } from "@db/DynamoBusinessDataClient";
import { DynamoDataClient } from "@db/DynamoDataClient";
import { DynamoUserDataClient } from "@db/DynamoUserDataClient";
import { BUSINESSES_TABLE, USERS_TABLE } from "@functions/config";
import { generateUserData } from "@shared/test";
import { CURRENT_VERSION } from "@shared/userData";
```